utils.js

```diff
@@ -165,3 +165,38 @@ class ObjectsMap {
         });
     }
 };
+
+let _currentDesktopsMatches = {};
```

Comment:
This can be `const` I guess

Reply:
(I know this is copy pasted code, but I didn't want to change the original commit since it's not made by us)